package.json

```diff
@@ -10,6 +10,7 @@
     "start": "openmrs develop",
     "serve": "webpack serve --mode=development",
     "build": "webpack --mode production",
+    "release": "lerna version --no-git-tag-version",
```

Comment:
This isn't a lerna repo is it? Is this going to work?

Reply:
Oops, no.